renderdoc/core/remote_server.cpp

```diff
@@ -584,6 +585,19 @@ static void ActiveRemoteClientThread(ClientThread *threadData,
         SERIALISE_ELEMENT(StackFrames);
       }
     }
+    else if(type == eRemoteServer_GetDriverName)
+    {
+      RDCLOG("Received eRemoteServer_GetDriverName request");
+      reader.EndChunk();
+
+      std::string driver = rdc ? rdc->GetDriverName() : "";
+      RDCLOG("Driver for the request is %s", driver.c_str());
```

Comment:
Are these log prints left over from debugging? I don't think we need them, but they should at least be demoted to `RDCDEBUG` to avoid too much log spam.

Reply:
Yes, forgot to remove them. Thanks for catching that!